tests/test_helpers.py

```diff
@@ -941,3 +942,38 @@ def test_safe_join_exceptions(self):
         for args in failing:
             with pytest.raises(NotFound):
                 print(flask.safe_join(*args))
+
+class TestHelpers(object):
+
+    def test_get_debug_flag(self, monkeypatch):
```

Comment:
Since this function is a bunch of asserts with different values for X in `monkeypatch.setenv('FLASK_DEBUG', X)`, couldn't you DRY this up using `pytest` parameterization?

Basically you can use a decorator to tell pytest "run this test 4 times with these 4 different sets of inputs and here are the 4 expected outputs".